types/index.d.ts

```diff
@@ -19,18 +19,21 @@ declare namespace Viewer {
     size?: ToolbarButtonSize,
   }
 
+  type ToolbarOption = boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions
+  
   export interface ToolbarOptions {
-    flipHorizontal?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    flipVertical?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    next?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    oneToOne?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    play?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    prev?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    reset?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    rotateLeft?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    rotateRight?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    zoomIn?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
-    zoomOut?: boolean | Visibility | ToolbarButtonSize | Function | ToolbarButtonOptions;
+    flipHorizontal?: ToolbarOption;
+    flipVertical?: ToolbarOption;
+    next?: ToolbarOption;
+    oneToOne?: ToolbarOption;
+    play?: ToolbarOption;
+    prev?: ToolbarOption;
+    reset?: ToolbarOption;
+    rotateLeft?: ToolbarOption;
+    rotateRight?: ToolbarOption;
+    zoomIn?: ToolbarOption;
+    zoomOut?: ToolbarOption;
+    [x: string]: ToolbarOption;
```

Comment:
this string index seems to have broken master for typescript apps.

![Screen Shot 2021-05-28 at 18 35 22](https://user-images.githubusercontent.com/4671809/120052743-f8f1a700-bfe3-11eb-8161-93b98a643b4a.png)

```
Error: node_modules/viewerjs/types/index.d.ts:23:5 - error TS2411: Property 'zoomOut' of type 'ToolbarOption | undefined' is not assignable to string index type 'ToolbarOption'.

23     zoomOut?: ToolbarOption;
```

Reply:
I can't reproduce the issue, [here](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcBqBLAzhgRhgGwyTgF44AGOAHzgEYa4AmRgZkYBYBuAKFElgJkqACoQIBXAEMoAIQCuMGBAB2AZQwAvVOQDkWALZSCBXY10HgAEwzyDZ2roIyA5sF29+0eEhRwxEtJQAPJgMBiqZHC44gTAUiqMmDj4RCS0AZIyCkqqGtqMAGLyKgDG4ZG0JVbAAGYYKta8fODeQn6ZQaEVKlhRAN48cMNwtURgABLQWqowxgD8AFz+sV1hESq8I6PjaMCwGKULy50y3RtbI40gMEsrgWfrqpfDqsBiwY13pyFPm0MjMDORDfVaPHovOBgKDAABuoIevwhAOGMKwwFuJzBSIuKLgUAgcxgwAAMnVMfcsjjnniCUTgAAlDAuAAWFJ+5xp2004gMAEkVAiqZz-tzecFFEK1sjtgBtEDLLAwKANFwAXSxiJFvAAvjweKVVEqEMAlZrhX8+uRBmKIAYJTBlsr5MAADR4qRYKxOqAu906oA). 

![5976280e-aaee-4061-8179-b10d10993685](https://user-images.githubusercontent.com/5789670/120059624-5aa12800-c05b-11eb-9e64-08f7c84d9597.gif)

@cevaris Do you mind creating a minimal repository with your current settings so that I can find what's causing that error?